File: C++/Kode/Tshirt.cpp
#ifndef TSHIRT_H //agar tidak terjadi tabrakan atau bahkan error, soalnya dari tadi error terus
#define TSHIRT_H

#include "Clothing.cpp"

class Tshirt : public Clothing {
private:
    string TipeBahan;

public:
    Tshirt(string brand, string size, double price, string TipeBahan) : Clothing(brand, size, price) {
        this->TipeBahan = TipeBahan;
    }
    
    void setTipebahan(string TipeBahan) { 
        this->TipeBahan = TipeBahan; 
    }

    string getTipebahan() { 
        return TipeBahan; 
    }
    
    void display() override {
        Clothing::display();
        cout << " | " << setw(15) << TipeBahan << " |" << endl;
    }
};

#endif

File: PYTHON/Kode/Tshirt.py
from Clothing import Clothing

class Tshirt(Clothing):
    def __init__(self, brand, size, price, sleeve_type):
        super().__init__(brand, size, price)
        self.sleeve_type = sleeve_type
    
    def get_sleeve_type(self): 
        return self.sleeve_type
    
    def set_sleeve_type(self, sleeve_type): 
        self.sleeve_type = sleeve_type
    
    def display(self):
        return super().display() + f" {self.sleeve_type:<13} |"